### PR TITLE
Add some details on how BMv2 v1model implements idle timeout option o…

### PR DESCRIPTION
**Pull request by jafingerhut**
…n tables

### CODE REVIEW

**Comment by jafingerhut** [comment]
Intended to resolve this issue: https://github.com/p4lang/behavioral-model/issues/1071

Antonin, note that some things I have written in this PR are my best guesses.  If you want me to do more double checking to verify those guesses, let me know.  If you do that double-checking, and have any corrections to make to my wrong guesses, I am happy to update this PR.

File: docs/simple_switch.md
Comment (antoninbas) [inline docs/simple_switch.md:1075]: technically it sweeps every second, but it won't send a notification for the same entry in 2 consecutive sweeps. I am not totally sure what the rationale was for implementing it this way... Maybe account for slower controllers and give them time to act as intended (delete the entry).

The end result is the same though: a notification is sent every 2 seconds until the entry is matched or the entry is deleted.

File: docs/simple_switch.md
Comment (antoninbas) [inline docs/simple_switch.md:1070]: I wonder if this doc should make it clear that only simple_switch_grpc exposes a P4Runtime API?

File: docs/simple_switch.md
Comment (jafingerhut) [reply docs/simple_switch.md]: I have attempted to make the text closer to your description of how things work.

File: docs/simple_switch.md
Comment (jafingerhut) [reply docs/simple_switch.md:1070]: I have added an explicit mention that only simple_switch_grpc implements the P4Runtime API, not simple_switch.

File: docs/simple_switch.md
Comment (antoninbas) [inline docs/simple_switch.md:1073]: We may want to add this to the beginning of the doc as a general disclaimer. I think you have been mentioning P4Runtime in a couple of other places. What do you think?

File: docs/simple_switch.md
Comment (antoninbas) [inline docs/simple_switch.md:1081]: that doesn't sound grammatically correct? maybe "and thus BMv2 has sent an IdleTimeoutNotification message to the controller for that entry"

File: docs/simple_switch.md
Comment (jafingerhut) [reply docs/simple_switch.md]: I have removed this sentence where you commented on it, and added a section near the beginning of the document that explicitly describes control plane API differences between simple_switch and simple_switch_grpc

File: docs/simple_switch.md
Comment (jafingerhut) [reply docs/simple_switch.md]: Edited to use your suggested wording, which does sound better.

File: docs/simple_switch.md
Comment (antoninbas) [inline docs/simple_switch.md:38]: This new section is great, but I think we should make it sure that Thrift is _not_ available by default in simple_switch_grpc to avoid confusion: https://github.com/p4lang/behavioral-model/tree/main/targets/simple_switch_grpc#enabling-the-thrift-server

Comment (jafingerhut) [reply docs/simple_switch.md]: Eventually you will teach me to say less, rather than more incorrect things, in my documentation, but not today :-). Thanks for the correction.

**Review by antoninbas — APPROVED** [review]
LGTM

**Comment by jafingerhut** [comment]
If you want this merged, I don't know who has approval to do so other than you, Antonin :-)

**Comment by antoninbas** [comment]
@jafingerhut I was just waiting for CI to complete, even though it's only a doc change :)

**Pull request merged**
